questionpy_sdk/_package/_builder.py

```diff
@@ -0,0 +1,237 @@
+
+                if po_file.with_suffix(".mo").exists():
+                    # By default, Poedit also saves a compiled .mo file, so we warn the user if one exists.
+                    _log.warning(
+                        "The existing .mo file at '%s' will not be used, '%s' will be compiled instead.",
+                        po_file.with_suffix(".mo"),
+                        po_file.name,
+                    )
+
+                outfile = tempdir / locale / i18n.DEFAULT_CATEGORY / f"{domain}.mo"
+                outfile.parent.mkdir(parents=True, exist_ok=True)
+
+                cmd = babel.messages.frontend.CompileCatalog()
+                cmd.locale = bcp47_to_posix(locale)
+                cmd.input_file = po_file
+                cmd.output_file = outfile
+                cmd.ensure_finalized()
+                cmd.run()
+
+            self._copy_glob(tempdir, "**/*.mo", self._target.dist / "locale", ignore_paths=False)
+
+    def _copy_glob(
+        self,
+        source_dir: Path,
+        glob: str,
+        dest_dir: Path,
+        *,
+        add_to_static_files: bool = False,
+        ignore_paths: PathSpec | bool = True,
+    ) -> None:
+        """Copy everything in `source_dir` matching `glob` to the same relative path under `dest_dir`.
+
+        If `add_to_static_files` is `True`, all copied files will be added to the static files list in the manifest.
+        `dest_dir` must be inside the dist dir in that case, or an error will be raised.
+
+        Will respect `self._ignore_spec` unless another PathSpec or `False` is passed in `ignore_path_spec`. `False`
+        will cause no files to be ignored.
+        """
+        if ignore_paths is True:
+            ignore_paths = self._ignore_spec
+
+        for source_file in source_dir.glob(glob):
+            dest_path = dest_dir / source_file.relative_to(source_dir)
+
+            if ignore_paths and ignore_paths.match_file(source_file.relative_to(source_dir)):
+                continue
+
+            _log.debug("%s: %s", dest_path, source_file)
+
+            if source_file.is_dir():
+                dest_path.mkdir(parents=True, exist_ok=True)
+            else:
+                dest_path.parent.mkdir(parents=True, exist_ok=True)
+                shutil.copy2(source_file, dest_path)
+
+            # register as static file in build manifest
+            if source_file.is_file() and add_to_static_files:
+                mime_type = guess_type(source_file)[0]
+                file_size = source_file.stat().st_size
+                path_in_dist = str(dest_path.relative_to(self._target.dist))
+                self._manifest.static_files[path_in_dist] = PackageFile(mime_type=mime_type, size=file_size)
+
+
+def build_qpy_package(source: PackageSource, target: BuildTarget | None = None, *, copy_sources: bool = True) -> None:
+    if not target:
+        from questionpy_sdk._package import DirBuildTarget  # noqa: PLC0415 (works around a circular import)
```

Comment:
```suggestion
from questionpy_sdk._package._targets import BuildTarget, DirBuildTarget
```

Der lokale import an dieser Stelle ist unnötig. Du kannst das als top-level import schreiben, indem du von `questionpy_sdk._package._targets` importierst.